keymaps/intellij-idea-keymap.cson

```diff
@@ -109,17 +136,55 @@
 
   # Go to line
   'cmd-l': 'go-to-line:toggle'
+  'cmd-\\': 'unset!' # cancel the tree-view shortcut
+  'cmd-k cmd-b': 'unset!' # cancel the tree-view shortcut
 
   # Recent files popup
   'cmd-e': 'fuzzy-finder:toggle-buffer-finder'
 
+  # Reveal active file
+  'cmd-ctrl-s': 'tree-view:reveal-active-file'
+
+  # Show bookmarks
+  'cmd-f3': 'bookmarks:view-all'
+
+  # Toggle bookmark
+  'f3': 'bookmarks:toggle-bookmark'
+
   ##################
   # Live Templates #
   ##################
 
   # Insert live templates
   'cmd-j': 'snippets:available'
 
+'.platform-darwin .tree-view':
+
+  ###########
+  # General #
+  ###########
+
+  'a': 'unset!'
```

Comment:
I would not unset the standard shortcuts as the shortcuts for idea should just be an addition to the default ones. Could you maybe remove all the `unset!` shortcuts?